info.json

```diff
@@ -9,56 +9,41 @@
             "name": "sOARFramework",
             "type": "solutionpack",
             "label": "SOAR Framework",
-            "version": "1.0.0",
+            "version": "1.1.0",
             "minVersion": null
         }
     ],
     "prerequisite": null,
-    "publisher": "Community",
-    "description": "Playbooks to extract file metadata and content. The result is added to the record's description",
-    "help": null,
+    "publisher": "Fortinet SE",
+    "description": "Playbooks to extract file metadata and content. The result is added to the record's description and notify the count of IOCs",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/README.md",
     "category": [
-        "Investigation",
-        "Enrichment"
+        "Enrichment",
```

Comment:
enrichment is a wrong category. It Extracts the IOC. @cs-rohit-khune  what should be the correct name here?

Reply:
IMO `Utilities` would be better @cs-suyog-jinturkar